### DISCUSSION TITLE
Avalonia build error

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [AvaloniaUI/Avalonia#6584] [Q&A]
Hi,

I am trying to build the `Avalonia.sln` locally on windows and everytime I am getting this error. Looking into this file it's an autogenerated one. I am not sure how to resolve this. Any help Is much appreciated. Thanks in advance.

![image](https://user-images.githubusercontent.com/79670853/133141435-8ca89e4a-0f35-4d91-a139-9b1d47587893.png)

**Comment by grokys** [comment]
Strange, we build the solution on every CI build so it should work. Have you tried using the `dotnet build` command line instead of VS? Sometimes VS gets confused and doing a command-line build often fixes it. Also try running a `git clean -xdf` before building.
